src/Hocon.Immutable/Extensions/GenericGetterExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hocon.Immutable.Extensions
+{
+    public static class GenericGetterExtensions
```

Comment:
Generic value getter implementation. Converts an immutable Hocon literal string value to another primitive type. Usage: `myHoconLiteral.Get<int>()`